.stylelintrc

```diff
@@ -3,5 +3,13 @@
     "stylelint-config-standard",
     "stylelint-config-idiomatic-order",
     "stylelint-config-prettier"
-  ]
+  ],
+  "rules": {
+    "block-opening-brace-newline-after": "always",
+    "block-opening-brace-space-before": "always",
+    "block-closing-brace-newline-after": "always",
+    "declaration-block-trailing-semicolon": "always",
+    "declaration-colon-space-after": "always",
+    "selector-list-comma-space-after": "always-single-line"
+  }
 }
```

Comment:
I thought that prettier was doing some or all of this so I'm a bit surprised. Does our prettier configuration work at all?

Reply:
I was also pretty surprised that it doesn't work. I saw it while reviewing https://github.com/firefox-devtools/profiler/pull/1915/files#diff-3f44625917b1301c0a0297e00d306a4fR35
Maybe it's broken or something since they can push like that?

Reply:
Actually I just tried to run `yarn lint-fix-css-prettier` and it changed a few css files. That means we don't have a hook for that. But I'm a bit confused how we manage all git hooks. It looks like we use husky but I don't see a config file for that.

Reply:
We don't fix automatically anymore as this was confusing for some users (including myself :) ). But we should check it at push time.
And now looking at our package.json I see that we don't do it, and only run stylelint on push (see the end of this comment for more information about husky).

So I think this is what we can do:
* current `lint-css` be renamed in `lint-css-stylelint`
* add a `lint-css-prettier` that would run prettier in check mode only
* add a `lint-css` that would be `run-p lint-css-prettier lint-css-stylelint`

Alternatively we can integrate prettier in stylelint. That's the approach we took for eslint + prettier and we can do the same for stylelint with [stylelint-prettier](https://github.com/prettier/stylelint-prettier) so that would be my prefered approach.

One note on husky:
Our version of husky runs the `prepush` script when pushing:
```
    "prepush": "yarn test-all",
```
We could upgrade to the newer version that has a configuration part (it had a compatibility problem but I fixed it some time ago in https://github.com/firefox-devtools/profiler/pull/1825).

Reply:
> Alternatively we can integrate prettier in stylelint. That's the approach we took for eslint + prettier and we can do the same for stylelint with stylelint-prettier so that would be my prefered approach.

I like this idea! I'll update the PR with this solution.